sql/moz-fx-data-shared-prod/firefox_desktop_derived/adclick_history_v1/query.sql

```diff
@@ -2,20 +2,15 @@ WITH new_data AS (
   SELECT
     client_id,
     sample_id,
-    ad_click AS ad_clicks
+    SUM(ad_click) AS ad_clicks
   FROM
     `moz-fx-data-shared-prod.search_derived.search_clients_daily_v8`
   WHERE
     submission_date = @submission_date
     AND ad_click > 0
-),
-existing_data AS (
-  SELECT
+  GROUP BY
     client_id,
-    sample_id,
-    ad_click_history
-  FROM
-    `moz-fx-data-shared-prod.firefox_desktop_derived.adclick_history_v1`
+    sample_id
```

Comment:
Each client_id is related to only one sample_id, is it still required?

Reply:
I just wanted to include it so that people could query by sample ID if they wanted to limit results - but to your point, if I took it out of the schema/query completely, the query would probably run faster?

Reply:
OK I ended up taking it out since really it just adds work to the query and I don't really need it in this table, trying to make this run as efficiently as possible - thanks for the suggestion!